ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/gmake/tasks/main.yml

```diff
@@ -85,3 +85,40 @@
     - /tmp/make-{{ makeVersion }}.tar.gz
   ignore_errors: yes
   tags: goodmake_source
+
```

Comment:
Could this be simplified down to two blocks:
```
if yum.conf does not exist
   create it with one line [main]
```
Then have the extra block to add `exclude=make` that gets executed regardless of whether it did the initial create or if it was already there?

Reply:
I can make it so it: (each bullet is a single 'task')
- Check if /etc/yum.conf exists
- Creates /etc/yum.conf with [main] in it, if it doesn't exist
- Put `exclude=make` into /etc/yum.conf 
Is that what you meant? It'd be the same amount of tasks

Reply:
Yes but it avoids duplicating the exclude line in two sections, so that is preferable. Please proceed :-)

Reply:
Fixed :-)